package com.voya.userapp.exception;

public class NameExistException extends Throwable {
    public NameExistException(String name_already_exist) {
    }



}
